packages/shared/src/components/auth/AuthenticationBanner.tsx

```diff
@@ -32,12 +38,18 @@ export function AuthenticationBanner(): ReactElement {
         srcSet={`${laptopBg} 1440w, ${desktopBg} 1920w, ${bg} 2880w`}
         sizes="(max-width: 1440px) 100vw, (max-width: 1920px) 1920px, 100vw"
       />
-      <Section className="w-[32.5rem]">
-        <OnboardingHeadline
-          className={{ title: 'typo-mega3', description: 'mb-8 typo-title3' }}
-        />
+      <Section className="w-[32.5rem] gap-4">
+        {children}
+        {!children && (
+          <OnboardingHeadline
+            className={{
+              title: 'typo-mega3',
+              description: 'mb-8 typo-title3',
+            }}
+          />
+        )}
```

Comment:
Would this not work? Not a blocker though.

```suggestion
        {children || (
          <OnboardingHeadline
            className={{
              title: 'typo-mega3',
              description: 'mb-8 typo-title3',
            }}
          />
        )}
```